teacher-guide/course_settings/learning_path_settings.md

```diff
@@ -1,19 +1,15 @@
 # Leerpad instellingen
 
-De sectie Leerpad ![](../../.gitbook/assets/graphics327%20%283%29.png) kan worden gebruikt om de _auto-lancering_ van leertrajecten en de stijl \(of grafisch thema\) voor de cursus mogelijk te maken.
+De sectie Leerpad ![](../../.gitbook/assets/graphics327%20%283%29.png) kan worden gebruikt om de *auto-lancering* van leertrajecten en de stijl (of grafisch thema) voor de cursus mogelijk te maken.
 
-![](../../.gitbook/assets/images246%20%283%29.png)
+![](../../.gitbook/assets/images246%20%283%29.png)Afbeelding 177: Cursusinstellingen - Leerpadinstellingen
 
 Afbeelding 177: Cursusinstellingen - Leerpadinstellingen
```

Comment:
There are two times the legend of this image now... You probably want to remove one.

In the other languages, we now put the legend two lines below, as this avoids a layout bug where the image appears on the left and the legend on the right, and because of this the image size is considerably reduced.

Also, in other languages we have now removed the image numbering ("Afbeelding 177: " becomes "Afbeelding: ") as the numbering was initially generated automatically by LibreOffice (and you wouldn't have to change all the numbering just to insert one new image in the middle). This doesn't make sense anymore.

We also have removed "page references" ("see page 177" for example) as now there is no page number.

Reply:
`We also have removed "page references" ("see page 177" for example) as now there is no page number.`
Could you give a me a file where this is still mentioned i could''t find it.